### DISCUSSION TITLE
Issue compiling DAC code for ESP32S3 V1.5.0 ESP-IDF

### DISCUSSION THREAD

**Discussion by AllenGovender** [Mair/esp32-course#195] [Course Qeustions]
I am at the GPIO section trying to compile the DAC code using the ESP32-S3-MINI-1 dev kit. It is coming up with a string of errors in the build process due to the DAC.H file not being found. I checked the "esp-idf\components\driver\esp32s3\include\driver" folder and found that the dac.h file is missing. Is there a specific dac.h file available for the esp32S3 processor?
Your help will be highly appreciated.
Thank you.

**Comment by noweare1** [comment]
You could try taking a look at the data sheet online at espressif.
